internal/provider/storage/storage_quota_rules_resource_test.go

```diff
@@ -0,0 +1,85 @@
+package storage_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	ntest "github.com/netapp/terraform-provider-netapp-ontap/internal/provider"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccStorageQuotaRulesResource(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { ntest.TestAccPreCheck(t) },
+		ProtoV6ProviderFactories: ntest.TestAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
```

Comment:
Can we have a test with the required parameters only if it makes sense?

Reply:
the current test covers all scenarios, so it should be fine

Reply:
ok. I just want to make sure if the user provides only required, very limited basic parameters and it should work.